interfaces/builtin/fwupd.go

```diff
@@ -148,6 +157,20 @@ const fwupdConnectedPlugAppArmor = `
       interface=org.freedesktop.DBus.Properties
       peer=(label=###SLOT_SECURITY_TAGS###),
 
+  dbus (send)
+      bus=system
+      path=/org/freedesktop/systemd1
+      interface=org.freedesktop.systemd1.Manager
+      member={GetDefaultTarget,GetUnit}
+      peer=(label=unconfined),
+
+  dbus (send)
+      bus=system
+      path=/org/freedesktop/login1
+      interface=org.freedesktop.login1.Manager
+      member=Reboot
+      peer=(label=unconfined),
```

Comment:
Reboot is already allowed via the 'shutdown' interface. fwupd should simply plugs that.

Reply:
Right, will remove this. Thanks.